style/themes/lcars.css

```diff
@@ -2188,3 +2188,10 @@ td.highlight {
   padding-top: 0.2em;
   padding-bottom: 0.55em;
 }
+
+/*** 🖖 ***/
+svg.fa-hand path {
+  d: path(
+    "M246.9 23.7C242.3 6.6 224.8-3.5 207.7 1.1s-27.2 22.1-22.6 39.2L238 237.8c2.5 9.2-4.5 18.2-14 18.2c-6.4 0-12-4.2-13.9-10.3L166.6 102.7c-5.1-16.9-23-26.4-39.9-21.3s-26.4 23-21.3 39.9l62.8 206.4c2.4 7.9-7.2 13.8-13.2 8.1L99.6 283c-16-15.2-41.3-14.6-56.6 1.4s-14.6 41.3 1.4 56.6L156.8 448c43.1 41.1 100.4 64 160 64l10.9 0 8.2 0c.1 0 .1-.1 .1-.1s.1-.1 .1-.1c58.3-3.5 108.6-43.2 125.3-99.7l81.2-275c5-16.9-4.7-34.7-21.6-39.8s-34.7 4.7-39.8 21.6L443.5 247.1c-1.6 5.3-6.4 8.9-12 8.9c-7.9 0-13.8-7.3-12.2-15.1l36-170.3c3.7-17.3-7.4-34.3-24.7-37.9s-34.3 7.4-37.9 24.7L355.1 235.1c-2.6 12.2-13.3 20.9-25.8 20.9c-11.9 0-22.4-8-25.4-19.5l-57-212.8z"
```

Comment:
Since there were many recent Pull Requests to improve image compression, I'm suggesting a very similar path, but using less characters (605 -> 318).

```suggestion
    "M247 24a32 32 0 1 0-62 16l53 198c3 9-4 18-14 18-6 0-12-4-14-10l-43-143a32 32 0 0 0-62 18l63 207c3 8-7 13-13 8l-55-53a40 40 0 0 0-56 58l113 107c43 41 100 64 160 64h19c58-4 109-43 125-100l82-275c5-17-5-35-22-40s-35 5-40 22l-37 128c-2 5-7 9-12 9-8 0-14-7-13-15l36-170c4-18-7-35-24-38s-35 7-38 24l-38 178a26 26 0 0 1-51 1z"
```

Reply:
The look the same to me (`display: none;` added at end of gif to prove it updates on the fly)

Voodoo magic

![no difference](https://github.com/user-attachments/assets/8bcf0be1-4dbd-474f-be85-9686e76a75ed)

Reply:
> The look the same to me...

Both paths are almost the same. The differences can only be seeing if you overlay both images and zoom in.

Reply:
Actually yeah - looks like ScreenToGif picked up a subtle difference when it was encoding. All good though, naked eye doesn't notice.